test/test_threaded.py

```diff
@@ -0,0 +1,115 @@
+#!/usr/bin/env python3
```

Comment:
There is a lot of duplicated code here. I think it should be sufficient to add a single new method to `test_fs.py`.

Reply:
I agree.
However most of the duplicated code due to the fact the vanilla version consider the fuse fs will be started in a separate process where here we use a thread (`wait_for_mount` takes a process as argument, `test_fs.Fs` takes a `cross_process`).
Do you think there is a way to mock those attributes or to make them optional ?

Reply:
There is good reason why the test is using a separate process, the risk of deadlocks is way too high. That applies to this test as well, just call stop() in the other process.

Reply:
I've reworked the test to use `test_fs.py`'s `testfs` fixture (I had to modify it a bit to make the process start a thread monitoring an event to call `stop`, given `multiprocessing.Event` cannot be serialized by `Manager.Namespace` I had to create a `need_stop` attribute along with `cross_process`)